src/Microsoft.VisualStudio.Editor.Razor/Documents/DefaultFileChangeTracker.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.VisualStudio.Editor.Razor.Documents
```

Comment:
There's going to be a lot lot lot more infrastructure soon oriented around this concern. This seems worth its own namespace.